data/sql_updates/large_aggregates.sql

```diff
@@ -142,11 +146,14 @@ party_totals as (
 -- Independent expenditure only
 ie as (
     select
-        extract(month from to_date(cast(coverage_end_date as text), 'YYYY-MM-DD')) as month,
-        extract(year from to_date(cast(coverage_end_date as text), 'YYYY-MM-DD')) as year,
-        sum(independent_expenditures_period) as receipts,
-        sum(independent_contributions_period) as disbursements
-    from ofec_reports_ie_only_mv_tmp
+        extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+        extract(year from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as year,
+        sum(ttl_indt_exp) as receipts,
+        sum(ttl_indt_contb) as disbursements
+    from
+        fec_vsum_f5
+    where
```

Comment:
I know it was probably mentioned somewhere so sorry if I'm slow on the uptake, but what is the rationale for recreating this logic here as opposed to just using the totals from `ofec_reports_ie_only_mv_tmp`?